test/adapters/flexibleLeverageStrategyAdapter.spec.ts

```diff
@@ -1130,6 +1148,7 @@ describe("FlexibleLeverageStrategyAdapter", () => {
         beforeEach(async () => {
```

Comment:
Can you add a test around here that tests when Chainlink oracles differ from Compound oracles, where Compound oracles are within the slippage limit, but Chainlink oracles are outside? We expect a `Slippage too high` revert as calculations use Chainlink vs Compound oracles. This is a sanity check to ensure our code is correct